src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1314,14 +1321,15 @@
                               (utils/pem->cert (path-to-cert signeddir subject))
                               (utils/pem->csr (path-to-cert-request csrdir subject)))
         default-fingerprint (fingerprint cert-or-csr "SHA-256")]
-    {:name          subject
-     :state         (certificate-state cert-or-csr crl)
-     :dns_alt_names (dns-alt-names cert-or-csr)
-     :fingerprint   default-fingerprint
-     :fingerprints  {:SHA1    (fingerprint cert-or-csr "SHA-1")
-                     :SHA256  default-fingerprint
-                     :SHA512  (fingerprint cert-or-csr "SHA-512")
-                     :default default-fingerprint}}))
+    {:name              subject
+     :state             (certificate-state cert-or-csr crl)
+     :dns_alt_names     (dns-alt-names cert-or-csr)
+     :subject_alt_names (subject-alt-names cert-or-csr)
```

Comment:
I didnt want to remove dns_alt_names from this payload, so I just added one that includes both IP AND DNS, which we can update our puppetserver-ca-cli to use.